plugins/modules/packaging/os/xbps.py

```diff
@@ -272,12 +277,12 @@ def main():
 
     module = AnsibleModule(
         argument_spec=dict(
-            name=dict(default=None, aliases=['pkg', 'package'], type='list'),
+            name=dict(default=None, aliases=['pkg', 'package'], type='list', elements='str'),
             state=dict(default='present', choices=['present', 'installed',
                                                    'latest', 'absent',
                                                    'removed']),
             recurse=dict(default=False, type='bool'),
-            force=dict(default=False, type='bool'),
+            force=dict(default=False, type='bool', removed_in='3.0.0', removed_from_collection='community.general'),
```

Comment:
Ah sorry, it must be `removed_in_version`. (I also made that mistake in https://github.com/ansible-collections/overview/issues/45#issuecomment-645619042 and just fixed it...)
```suggestion
            force=dict(default=False, type='bool', removed_in_version='3.0.0', removed_from_collection='community.general'),
```